app/views/homepage/popular_links/_sidebar.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="sidebar__button sidebar__border">
```

Comment:
I think I would assign this container a single class name (like "sidebar-components") to keep with the existing naming convention and use that to add custom CSS.